src/modules/vtol_att_control/vtol_type.cpp

```diff
@@ -286,13 +286,17 @@ bool VtolType::isUncommandedDescent()
 	    && (current_altitude < _tecs_status->altitude_reference)
 	    && hrt_elapsed_time(&_tecs_status->timestamp) < 1_s) {
 
+		if (!PX4_ISFINITE(_quadchute_ref_alt)) {
+			_quadchute_ref_alt = current_altitude;
```

Comment:
@sfuhrer I have used NAN to make `_quadchute_ref_alt` invalid when the logic should not run, that's why I first need to init it there. I used to use `-FLT_MAX` but that brings another challenge when trying to add the delta z for the reset, since you need to make sure that then the variable does not underflow.
Eventually, I decided that it makes more sense to use NAN.

Reply:
okay thanks for the explanation, makes sense!